sale_stock_available_info_popup/models/sale_order.py

```diff
@@ -18,9 +18,15 @@ def _compute_immediately_usable_qty_today(self):
         qty_processed_per_product = defaultdict(lambda: 0)
         self.immediately_usable_qty_today = False
         for line in self.sorted(key=lambda r: r.sequence):
+            if line.order_id.commitment_date:
+                date = line.order_id.commitment_date
```

Comment:
What is making the decision to take the commitment date and not the order date?

Shouldn't both computations in different fields ?

Reply:
Hi, @rousseldenis It comes from core: https://github.com/odoo/odoo/blob/13.0/addons/sale_stock/models/sale_order.py#L243-L246

Reply:
Could you add a test with commitment_date ?

Reply:
Added test for that condition and simplified test declaration using `Form`